flink-runtime/src/main/java/org/apache/flink/runtime/clusterframework/BootstrapTools.java

```diff
@@ -290,7 +291,8 @@ private static Config getExecutorConfigByExecutorMode(Configuration configuratio
 			case FORK_JOIN_EXECUTOR:
 				return AkkaUtils.getForkJoinExecutorConfig(configuration);
 			case FIXED_THREAD_POOL_EXECUTOR:
-				return AkkaUtils.getThreadPoolExecutorConfig();
+				return AkkaUtils.getThreadPoolExecutorConfig(
+					configuration.getInteger(MetricOptions.QUERY_SERVICE_THREAD_PRIORITY));
```

Comment:
seems quite odd to couple this method to the MetricOptions.

Reply:
Good point. Only the thread priority should be passed to this method here. I guess we need to introduce a executor config instead of simply using an enum.

Reply:
I will address this change before merging.

Reply:
OK, I agree with it. Then I will not change it anymore.